mkdocs/docs/HPC/VScode.md

```diff
@@ -0,0 +1,46 @@
+# VS Code Tunnel
+
+Please keep in mind, that this is not a VS Code manual,
+it is only our recommendation how to connect to the {{ hpcinfra }} using the VS code tunnel. 
+
+For VS Code documentation, see <https://code.visualstudio.com/docs>
+
+We only support VS Code tunnel access via the VS Code application (installed locally),
+since web access might not work from all browser or operational system.
+The only confirmed way to connect via a browser is using Windows 11 and Edge. 
```

Comment:
This neds to be clarified: this is the to connect to the tunnel URL via a browser